keras_nlp/src/utils/preset_utils.py

```diff
@@ -99,13 +113,15 @@ def get_file(preset, path):
         )
     if preset in BUILTIN_PRESETS:
         preset = BUILTIN_PRESETS[preset]["kaggle_handle"]
-    if preset.startswith(KAGGLE_PREFIX):
+
+    scheme = preset.split("://")[0].lower()
```

Comment:
Should we handle the filename case more explicitly?

```
sheme = None
if "://" in preset:
    scheme  = this stuff
```

Clearer, and handles super niche cases better, like someone making a directory named `kaggle` and trying to load from it.